src/minimal/a-nbnbin.adb

```diff
@@ -0,0 +1,8 @@
+
```

Comment:
Is there a reason for having an empty first line in these files?

Reply:
Fixed by https://github.com/Componolit/ada-runtime/pull/77/commits/1694bd3dfa66c1ad7748a6c6fe644599844427b6.